apps/vs-code-designer/src/app/commands/GenerateStaticWebApp/generateStaticWebApp.ts

```diff
@@ -0,0 +1,11 @@
+import type { IAzureConnectorsContext } from '../workflows/azureConnectorWizard';
+import type * as vscode from 'vscode';
+import { executeCommand } from '../../utils/funcCoreTools/cpUtils';
+import { ext } from '../../../extensionVariables';
+
+export async function generateStaticWebApp(context: IAzureConnectorsContext, node: vscode.Uri): Promise<void> {
+  console.log('generate SWA');
+  //removing "/c: because that doesn't fit SWA cli format"
+  const command: string = `node C:\\Users\\t-azhiyanov\\static-web-apps-cli-sifat\\dist\\cli\\bin.js generate ${node.path.substring(3)}`;
```

Comment:
how can this be a static path? Won't it fail to find the bits on another machine?

Reply:
@alain-zhiyanov I am not fully aware of the implementation or how we get this script, however if this is a static script, please upload the script in the repo or even better just make a function to call it. If its not static (generated), please make sure to have it a kind of static place for all users to work and also use `path.join()` since the pat diff for windows (\\) is different than the one for linux (/)